plugins/dashboard/frontend/src/app/components/ExplorerAddressResult.tsx

```diff
@@ -48,7 +48,7 @@ export class ExplorerAddressQueryResult extends React.Component<Props, any> {
                 msgsEle.push(
                     <ListGroup.Item key={msg.id}>
                         <small>
-                            {dateformat(new Date(msg.timestamp * 1000), "dd.mm.yyyy HH:MM:ss")} {' '}
+                            {dateformat(new Date(msg.solidification_timestamp * 1000), "dd.mm.yyyy HH:MM:ss")} {' '}

```

Comment:
I don't think we need any solidification timestamp in the ExplorerAddress no?

Reply:
That's the messages residing on the address but whatever, this code is going to be removed by your changes anyway.